src/server.ts

```diff
@@ -12,6 +12,11 @@ import exceptionHandler from './middlewares/exceptionMapper'
 const app = express()
 const server = http.createServer(app)
 
+app.use(function(req, res, next){
```

Comment:
Generalnie to wszystko poprawnie ale : 
* zamist normalnej sygnatury funkcji używaj, funkcji strzałkowych
* piszemy w `TypeScripcie` więc parametry funkcji powinny mieć określony typ
* nie trzymmay w pliku `server.ts` żadnej nie potrzebnej logiki dlatego proszę wyciągnij tą funkcję do dedykowanego pliku w folderze `middlewares`
* umieć instalację tego middleware-a za instalacją `corsów`